src/main/java/com/code_intelligence/jazzer/junit/FuzzTestArgumentsProvider.java

```diff
@@ -128,7 +132,10 @@ private Stream<? extends Arguments> adaptInputsForFuzzTest(
         Object[] args;
         if (argumentsMutator.isPresent()) {
           ArgumentsMutator mutator = argumentsMutator.get();
-          mutator.read(new ByteArrayInputStream(e.getValue()));
+          boolean readCompletely = mutator.read(new ByteArrayInputStream(e.getValue()));
+          if (!readCompletely) {
```

Comment:
This could also be called `readExactly`.

Reply:
Renamed.